CMakeLists.txt

```diff
@@ -12,9 +12,8 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
 	# Set the possible values of build type for cmake-gui
 	set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug" "Release" "MinSizeRel" "RelWithDebInfo")
 endif()
-message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
 
-project(IFCPP)

```

Comment:
I've merged this with previous `project`. There's no reason to do this twice with different names.